src/openrct2/title/TitleSequence.cpp

```diff
@@ -526,29 +527,30 @@ static std::string LegacyScriptWrite(const TitleSequence& seq)
             case TitleScript::EndLoop:
                 break;
             case TitleScript::Location:
-                String::Format(buffer, sizeof(buffer), "LOCATION %u %u", command.Location.X, command.Location.Y);
-                sb.Append(buffer);
+                formattedMessage = String::Format(
```

Comment:
No need for this temporary just pass it directly into append like you did in `UiContext.Linux.cpp`